packages/engine/lib/memory/src/arrow/arrow_batch.rs

```diff
@@ -0,0 +1,343 @@
+use std::sync::Arc;
```

Comment:
I think we can just call this module `batch`, I'd be inclined to still call the struct `ArrowBatch`.

Don't feel strongly about it though

Reply:
Yeah, absolutely. There is even a clippy lint for this, I think it's called something like "stuttering".